config/routes.rb

```diff
@@ -14,7 +14,14 @@
     end
   end
 
+  resources :course_instances, only: [] do
+    collection do
+      get :autocomplete
+    end
+  end
```

Comment:
If we use that syntax, we don't automatically get the canonical rails name for the path (:autocomplete_course_instances_path). To get that, we would have to do:

```get 'course_instances/autocomplete' => 'course_instances#autocomplete', as: :autocomplete_course_instances```